fawltydeps/cli_parser.py

```diff
@@ -171,6 +171,15 @@ def populate_parser_paths_options(parser: argparse._ActionsContainer) -> None:
             " Python environment where FawltyDeps is installed."
         ),
     )
+    parser.add_argument(
+        "--install-deps",
+        dest="install_deps",
+        action="store_true",
+        help=(
+            "Allow FawltyDeps to `pip install` declared dependencies into a"
+            " separate temporary virtualenv to discover the imports they expose."
+        ),
```

Comment:
I really like this help text. It's short and explains exactly what happens in a way that helps point the user in the right direction if they need more help. E.g. a user that that needs to customize how/where packages are downloaded now knows that `pip install` is what they need to customize. 👏🏻

Reply:
I agree :D I copied that from https://github.com/tweag/FawltyDeps/issues/266 😆

Reply:
😳 😆